apps/datahub/src/app/home/search/search-filters/search-filters.component.ts

```diff
@@ -14,6 +14,16 @@ import {
 } from '@geonetwork-ui/feature/search'
 import { getOptionalSearchConfig } from '@geonetwork-ui/util/app-config'
 
```

Comment:
I feel like this could even go in the fields.service.ts file, so that there is only one file to change when adding a new field